File: benchmarks/corpus/javascript/sqli_concat.js
const express = require("express");
const app = express();

app.get("/user", function (req, res) {
  const name = req.query.name;
  db.query("SELECT * FROM users WHERE name = '" + name + "'", function (err, rows) {
    res.json(rows);
  });
});

File: benchmarks/corpus/go/sqli_concat.go
package main

import (
	"database/sql"
	"net/http"
)

func user(w http.ResponseWriter, r *http.Request) {
	name := r.URL.Query().Get("name")
	row := db.QueryRow("SELECT * FROM users WHERE name = '" + name + "'")
	_ = row
	_ = sql.ErrNoRows
}
